scripts/compactor_runner.py

```diff
@@ -125,9 +121,12 @@ def append_compactor_config(self, compactor_config):
 
     def get_corfu_compactor_cmd(self, compactor_config, class_to_invoke):
         diskBacked = False
+        standardDiskBackedConfig = False
         if "DiskBacked" in compactor_config["MemoryOptions"] and compactor_config["MemoryOptions"]['DiskBacked'] is True:
             diskBacked = True
-        xmx = self.derive_xmx_value(diskBacked, compactor_config)
+        if "StandardDiskBackedConfig" in compactor_config["MemoryOptions"] and compactor_config["MemoryOptions"]['StandardDiskBackedConfig'] is True:
+            standardDiskBackedConfig = True
+        xmx = self.derive_xmx_value(diskBacked, standardDiskBackedConfig, compactor_config)
```

Comment:
nit: change names to `is_disk_backed` and `use_std_disk_backed_config` for better readability.